pkg/steps/build_defaults.go

```diff
@@ -42,7 +42,8 @@ func FromConfig(config *api.ReleaseBuildConfiguration, jobSpec *JobSpec, cluster
 
 	jobNamespace := jobSpec.Namespace()
 
-	var buildClient buildclientset.BuildInterface
+	var buildClient BuildClient
```

Comment:
Where is `BuildClient` defined? Why can't we use the upstream client?

Reply:
in steps, upstream doesn't handle logs, i don't want to go fix logs right now with ben out.

Reply:
Doh, forgot to push the file...